.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "semi": false,
+    "trailingComma": "none",
+    "singleQuote": false,
+    "printWidth": 80,
+    "quoteProps": "consistent"
+}
```

Comment:
These rules are my own, loosely-held preferences. Happy to modify now, but would hope that we reduce the amount of future modification to reduce noisy git history.

Reply:
It would make sense to align against existing SDKs on the assumption they too have a unified format. @adamjmcgrath ?

Reply:
That would make sense assuming folks are happy with those rules.